core/build.gradle

```diff
@@ -66,6 +66,7 @@ dependencies {
     implementation "com.squareup.okio:okio:$okioVersion"
     implementation "de.greenrobot:eventbus:$eventbusVersion"
     implementation "io.reactivex:rxandroid:$rxAndroidVersion"
+    implementation 'org.awaitility:awaitility:3.1.2'
```

Comment:
Every other version number is defined in the root gradle file ;)

Reply:
it is more commonly used for testing. But I don't see why we shouldn't use it here

Reply:
It looks so inconsistent this way. Specifying it in one single place could be useful in case we happen to add the same library to the app module in the future.